src/vs/workbench/contrib/notebook/browser/contrib/find/media/notebookFind.css

```diff
@@ -19,3 +19,7 @@
 	padding: 0 2px;
 	box-sizing: border-box;
 }
+
+.monaco-workbench .nb-findScope {
```

Comment:
maybe `nb-find-scope`

Reply:
editor uses the class `.findScope` so was just adding the same `nb-` prefix that we've used before. Can change this easily.